### DISCUSSION TITLE
HBase 1 removed in 0.6.2?

### DISCUSSION THREAD

**Discussion by timrobertson100** [JanusGraph/janusgraph#3365] [Q&A] [answered]
Hi folks

I struggled to get things running on HBase 1 with 0.6.2. I was able to build 0.6.1 using the `hbase1` profile which worked, but that is seemingly gone in 0.6.2.

Is it possible to run 0.6.2+ on HBase 1, please? If so, how?

**Comment by farodin91** [comment, marked as answer]
We didn't change anything between 0.6.1 and 0.6.2, so it should work fine. We will drop support for hbase 1.

**Comment by timrobertson100** [comment]
Thanks, @farodin91 

I see that now - sorry, my mistake. I must have been looking at `master` where it seems already removed [here](https://github.com/JanusGraph/janusgraph/blob/master/pom.xml#L70)
